<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminStaffChange</class>
 <widget class="QWidget" name="adminStaffChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>201</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>员工信息管理</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;宋体&quot;;</string>
   </property>
   <property name="text">
    <string>请输入要修改的员工工号：：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSno">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuery">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChange">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(255,255,255);
		border: 2px solid #b0b0b0;
       background-color: rgb(0, 0, 255);
		border-radius:15px;
		min-width:80px
}
QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(0, 0, 170);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(0,0,255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>重新查询</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>300</width>
     <height>350</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>员工信息</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>下班时间：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combDepart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择部门</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>行政部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>工程部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>技术部</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>销售部</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>工作状态：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combState">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>请选择工作状态</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>正常上班</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>休假</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>出差</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineSname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>部门：</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="amTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="pmTime">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="displayFormat">
     <string>HH:mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>上班时间：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>250</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>员工帐号</string>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>设置问题：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUserQue">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>问题答案：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUserAns">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>设置为管理员：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combUserIsAdmin">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>否</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>是</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineUserPS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0,0,0);
border: 1px solid #000000;
background-color: rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>修改密码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>84</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(0,0,0);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 255, 255);
		border-radius:15px;
		min-width:80px
}QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(200, 200, 200);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
		color:rgb(255,255,255);
		border: 2px solid #b0b0b0;
background-color: rgb(255, 0, 0);
		border-radius:15px;
		min-width:80px
}
QPushButton:hover{ 
color:rgb(255, 255, 255);
background-color: rgb(150, 0, 0);
border:2px solid  #b0b0b0;
border-radius:15px;
}
QPushButton:pressed{
color:rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border:2px solid  #b0b0b0;
border-radius:15px;
}</string>
   </property>
   <property name="text">
    <string>删除员工</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
